### PR TITLE
feat(JBP-1): add approver

### PR DESCRIPTION
**Pull request by thirdygayares**
approve

- screesnhjoty

image

### CODE REVIEW

**Review by rodney-lqr — COMMENTED** [review]
Improve this :)
Redundant
Follow naming convention
it seems complicated

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,336 +1 @@
...
-<!-- * * * * * * * * * * * The content above * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * is only a placeholder * * * * * * * * * * * -->
-<!-- * * * * * * * * * * and can be replaced.  * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * End of Placeholder  * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-
-
-<router-outlet />
+[JBP-1] - User Management Module Approved Company/Employer
```
Comment (thirdygayares) [inline src/app/app.component.html:1]: test

Comment (thirdygayares) [inline src/app/app.component.html:1]: ankter

Comment (rodney-lqr) [inline src/app/app.component.html:1]: test(2)

**Review by rodney-lqr — CHANGES_REQUESTED** [review]

File: src/app/app.component.html (hunk 1 above)
Comment (thirdygayares) [reply src/app/app.component.html:1]: sure boss

**Pull request merged**